torq.sh

```diff
@@ -234,19 +233,27 @@ else
 fi
 
 if [[ "$1" == "start" ]]; then
-  if [[ $(echo ${BASH_ARGV[*]} | grep -e print) ]]; then         
-    checkextrascsv "${*%${!#}}";
-    for p in $PROCS;
-    do 
-      print "$p";  
-    done
-  else 
-    checkextrascsv "$*";
-    startprocs "$PROCS";
-  fi
+  checkextrascsv "$*";
+  startprocs "$PROCS";
+elif [[ "$1" == "print" ]]; then         
+  checkextrascsv "$*";
+  for p in $PROCS;
```

Comment:
This do something for all procs is often repeated, something that could be turned into a function? @jonnypress @jonathonmcmurray

Reply:
I'm not sure how easy that is, my bash-fu isn't the strongest...can you pass a function as an argument to another function? I think that'd be required here, essentially pass the list and the function to run on list. Could possibly be done passing the name of the function & using `eval` but `eval` makes me nervous

Reply:
Actually, it can be done without `eval`

```bash
jmcmurray@homer ~ $ function f() { echo $1; }
jmcmurray@homer ~ $ function each() { for x in $2; do $1 $x; done; }
jmcmurray@homer ~ $ each f "a b c d"
a
b
c
d
```

Reply:
Great! Even though I brought this up I think that should be moved to another branch so as not to derail this one.